content/geoip/docs/databases/anonymous-ip.md

```diff
@@ -0,0 +1,165 @@
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163-Anonymizer-and-Proxy-Data#h_01FN9BBGV3ZG3TC0357Q9Y07C6" target="_blank" rel="nofollow noopener noreferrer">Learn more about anonymizer and proxy detection on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_anonymous_vpn</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address is registered to an anonymous VPN provider. Blank if not.
+
+            If a VPN provider does not register subnets under names associated with
+            them, we will likely only flag their IP ranges using the
+            <code>is_hosting_provider</code> flag.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163#h_01G1EDVJKNQY02XXFRM31N7SS2" target="_blank" rel="nofollow noopener noreferrer">Learn more about VPNs on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_hosting_provider</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address belongs to a hosting or VPN provider (see
+            description of <code>is_anonymous_vpn</code> flag). Blank if not.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163#h_01G1EDVXR9RQFMCY6SAWJM4YH0" target="_blank" rel="nofollow noopener noreferrer">Learn more about hosting providers used for anonymizing on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_public_proxy</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address belongs to a public proxy. Blank if not.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163#h_01G1EDW5RZQCD4X4A76908DJ6Z" target="_blank" rel="nofollow noopener noreferrer">Learn more about public proxies on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_tor_exit_node</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address is a Tor exit node. Blank if not.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163#h_01G1EDWR1VQR8A0NB3R8WXT8QF" target="_blank" rel="nofollow noopener noreferrer">Learn more about Tor exit nodes on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+      <tr>
+          <td>is_residential_proxy</td>
+          <td>boolean</td>
+          <td>
+            1 if the IP address is on a suspected anonymizing network and belongs
+            to a residential ISP. Blank if not.
+            <br />
+            <a href="https://support.maxmind.com/hc/en-us/articles/4408208507163#h_01G1EDWEW3HAJT97P8EAZFQZMZ" target="_blank" rel="nofollow noopener noreferrer">Learn more about residential proxies on our Knowledge Base.</a>
+          </td>
+      </tr>
+
+    </tbody>
+  </table>
+</div>
+{{</ rawhtml >}}
+
+### Conversion Utility
```

Comment:
I wonder if we should start including these headers within the snippets as well to reduce code duplication. Or is this format easier to read?

Reply:
I was torn about this.  To me, it's easier to see the overall structure of the doc when keeping the headers outside of the snippets.